dags.yaml

```diff
@@ -497,7 +497,7 @@ bqetl_search_dashboard:
     retries: 2
     retry_delay: 30m
     start_date: "2020-12-14"
-  schedule_interval: 30 4 * * *
+  schedule_interval: 30 5 * * *
```

Comment:
Just to double check, the execution_delta in the generated version of this DAG will get automatically updated to match the upstream DAGs execution date?

Reply:
Yes